flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
Is the python stuff the main thing holding off aarch64?

Reply:
`flatpak-pip-generator --runtime="org.kde.Sdk//6.6" --ignore-pkg 'Pyside6' -r requirements.txt --yaml` will give you everything in that file except Pyside6. You can either build pyside from source (complicated) or use wheels for that one only


A bunch of the dependencies like scipy and numpy require other dependencies to be available for source builds. You can follow this.

Reply:
Yes, just the dependencies I'm pretty sure.
I tried applying those extra steps but I'm kinda stuck trying to get it to work again on x86, let alone arm.
The issue now is:

```
Running: pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "opencv-python" --no-build-isolation
Using pip 23.3.1 from /usr/lib/python3.11/site-packages/pip (python 3.11)
Looking in links: file:///run/build/python3-opencv-python
Processing ./opencv-python-4.9.0.80.tar.gz
  Running command Preparing metadata (pyproject.toml)
  Traceback (most recent call last):
    File "/usr/lib/python3.11/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
      main()
    File "/usr/lib/python3.11/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3.11/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 149, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3.11/site-packages/setuptools/build_meta.py", line 366, in prepare_metadata_for_build_wheel
      self.run_setup()
    File "/usr/lib/python3.11/site-packages/setuptools/build_meta.py", line 480, in run_setup
      super(_BuildMetaLegacyBackend, self).run_setup(setup_script=setup_script)
    File "/usr/lib/python3.11/site-packages/setuptools/build_meta.py", line 311, in run_setup
      exec(code, locals())
    File "<string>", line 10, in <module>
  ModuleNotFoundError: No module named 'skbuild'
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /usr/bin/python /usr/lib/python3.11/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpjdvp74o_
  cwd: /tmp/pip-install-yvlcuhs4/opencv-python_4d08b27f5faa4d06a12b83a9a6356cd9
  Preparing metadata (pyproject.toml) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Error: module python3-opencv-python: Child process exited with code 1
make: *** [Makefile:14: build-install] Error 1
```
But adding scikit-build to the dependencies and also installing it locally didn't help. I'm at a loss. 

The current commit is broken now, I don't know if I can resolve this. This is getting more and more messy.
Reverting it would at least provide x86 support. Anyone on arm could at least use pip to install.

Reply:
Looks like torch in your dependency list depends upon a bunch of NVIDIA CUDA modules, many of which doesn't provide aarc64 or source packages eg. https://pypi.org/project/nvidia-nvjitlink-cu12/#files

I guess you have to restrict it to x86_64 only at the moment.

Reply:
Oh, that would explain why it can't find what it needs, that's unfortunate.

I can revisit this once the situation with torch improves. Reverting the broken commit for now.